scripts/ceph-windows/collect-event-logs.ps1

```diff
@@ -0,0 +1,45 @@
+param (
+    [Parameter(Mandatory)]
+    [string]$LogDirectory,
+    [switch]$IncludeEvtxFiles = $false,
+    [switch]$CleanupEventLog = $false
+)
+
+function DumpEventLogEvtx($path){
+    foreach ($i in (Get-WinEvent -ListLog * |  ? {$_.RecordCount -gt 0 })) {
+        $logName = "eventlog_" + $i.LogName + ".evtx"
+        $logName = $logName.replace(" ","-").replace("/", "-").replace("\", "-")
+        Write-Host "exporting "$i.LogName" as "$logName
```

Comment:
I'd suggest using `Write-Output` instead of `Write-Host` in this script. See [here](https://stackoverflow.com/questions/19754069/powershell-difference-between-write-host-and-write-output#:~:text=In%20a%20nutshell%2C%20Write%2DHost,is%20implicitly%20called%20for%20you.).

`Write-Host` writes to the console (which cannot be captured / redirected), not to the standard output.

When writing scripts, it's a good practice to use `Write-Output`. Because, if somebody needs to redirect the output of the script, it won't be able to capture anything written with `Write-Host`.